invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html

```diff
@@ -35,24 +35,28 @@
 {% macro show_creatibutors(creatibutors, show_affiliations=False) %}
   {% for creatibutor in creatibutors %}
   <dd class="creatibutor-wrap">
-    {{ creatibutor_icon(creatibutor) }}
-    <a class="ui tooltip-popup text-muted creatibutor-link"
+    <a class="ui tooltip-popup creatibutor-link"
       {% if show_affiliations and creatibutor.affiliations %}
         data-content="{{ creatibutor.affiliations|join('; ', attribute='1') }}"
       {% endif %}
       href="{{url_for('invenio_search_ui.search', q='metadata.creators.person_or_org.name:"' + creatibutor.person_or_org.name + '"')}}"
     >
 
       <span class="creatibutor-name">
-        {{ creatibutor.person_or_org.name }}
+        {{- creatibutor.person_or_org.name -}}
       </span>
-
-      <sup class="font-tiny text-muted">
-        {% for affiliation in creatibutor.affiliations %}
-          {{ affiliation[0] }}{{ ", " if not loop.last }}
-        {% endfor %}
+      {%- if creatibutor.affiliations -%}
+      <sup class="font-tiny">
+        {%- for affiliation in creatibutor.affiliations -%}
+          {{- affiliation[0] -}}{{- ", " if not loop.last -}}
```

Comment:
For Jinja interpolation `{{-` should typically not be needed. It can actually be confusing: value negated or space ignored? For `{%-` that's fine and typically nice.